### PR TITLE
TE-1818 Footer can accept no props.languageOptions and props.currencyOptions

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-1818)

### What **one** thing does this PR do?
Removes the language and currency `Submenu` from the `Footer` when `languageOptions` and `currencyOptions` aren't passed; then, updates the mentioned cases in the docs. 

### Behaviour
screenshot 2019-02-06 at 17 34 25

### CODE REVIEW

File: src/components/collections/Footer/component.js [hunk 1]
```diff
@@ -54,26 +54,30 @@ export const Component = ({
...
-            isSearchable
-            items={currencyOptions}
-            name="currency"
-            noResultsText={currencyNoResultsText}
-            onChange={onChangeCurrency}
-            value={currencyValue}
-            willOpenAbove
-          />
-        </Menu.Item>
+        {size(languageOptions) > 0 && (
```
Comment (marcdavi-es) [inline src/components/collections/Footer/component.js]:
```suggestion
        {size(languageOptions) > 1 && (
```

right?

File: src/components/collections/Footer/component.js [hunk 2]
```diff
@@ -54,26 +54,30 @@ export const Component = ({
...
+            <Submenu
+              items={languageOptions}
+              name="language"
+              onChange={onChangeLanguage}
+              value={languageValue}
+              willOpenAbove
+            />
+          </Menu.Item>
+        )}
+        {size(currencyOptions) > 0 && (
```
Comment (marcdavi-es) [inline src/components/collections/Footer/component.js]:
```suggestion
        {size(languageOptions) > 1 && (
```

Comment (Jonnygwi) [reply src/components/collections/Footer/component.js]: I think Marc is right. If there is only one option then there isn't any other to choose between :)

**Comment by Jonnygwi** [comment]
Removing myself as an assignee as I agree with Marc's requested changes.

**Pull request merged**